template-helpers/breach-detail.js

```diff
@@ -14,7 +14,7 @@ function getVars(args) {
 // How to dynamically detect Data Aggregator breaches?
 
 function getBreachCategory(breach) {
-  if (["Exactis"].includes(breach.Name)) {
+  if (["Exactis", "Apollo", "YouveBeenScraped", "ElasticsearchSalesLeads", "Estonia", "MasterDeeds"].includes(breach.Name)) {
```

Comment:
I personally think Adapt, Verifications.io, Data & Leads, and others are all data aggregators. How comprehensive do we want/need this list to be? (cc @betsymi )

Reply:
I prefer to err on the side of comprehensive. This makes our current data aggregator breach list:

```js
  "dataAggregatorBreaches": [
    "Adapt",
    "Apollo",
    "DataAndLeads",
    "ElasticsearchSalesLeads",
    "Estonia",
    "Exactis",
    "MasterDeeds",
    "VerificationsIO",
    "YouveBeenScraped"
  ],
```

Reply:
@pdehaan @groovecoder Thank you for the work you've done to aggregate the data aggregators.